docs/user/metal/install_ipi.md

```diff
@@ -0,0 +1,3 @@
+# Install: Baremetal Installer-Provided Infrastructure (IPI)
+
+TBD.
```

Comment:
I figure it's easier to collaborate on this in Google docs - will convert to markdown once we're happy w/ it.

https://docs.google.com/document/d/1r4MzBm2J_vMU42g6LDiLtCdfMI_2sgf7A7HbFVOTz3w/edit?usp=sharing

Reply:
Looks like there hasn't been ongoing collaboration on this - I think what you have is good enough to merge and include in the next upstream PR refresh

Reply:
It looks like the README fixes already went in, and most of the rest is in a google doc.  I wanted to work more on user docs today.  I'll incorporate the contents of the google doc and get a new PR opened later today.